### PR TITLE
SOC-2441 | top margin for the post list on mobile on post-details and…

### PR DESCRIPTION
**Pull request by rafalkalinski**
## Links

https://wikia-inc.atlassian.net/browse/SOC-2441

## Description

Back to All Discussions on Mobile Web Has a UI Bug

## Reviewers
@kvas-damian

### CODE REVIEW

**Comment by kvas-damian** [comment]
LGTM

File: front/main/app/styles/component/_discussion-side-button.scss [hunk 1]
```diff
@@ -1,13 +1,13 @@
 .side-button {
 	align-items: center;
 	display: flex;
-	font-size: 0.75rem;
-	font-weight: bold;
+	font-size: $type-smallest;
+	font-weight: $font-weight-normal;
```
Comment (kvas-damian) [inline front/main/app/styles/component/_discussion-side-button.scss]: Are you sure we should change this font-size?

Comment (rafalkalinski) [inline front/main/app/styles/component/_discussion-side-button.scss]: Derek is. That's enough for me :)

Comment (kvas-damian) [inline front/main/app/styles/component/_discussion-side-button.scss]: :+1:

**Pull request merged**